lib/cuckoo/core/startup.py

```diff
@@ -487,6 +487,28 @@ def cuckoo_clean():
         except:
             log.warning("Unable to drop MongoDB database: %s", mdb)
 
+    # Check if ElasticSearch reporting is enabled and drop that if it is.
+    cfg = Config("reporting")
+    if cfg.elasticsearch and cfg.elasticsearch.enabled:
+        from elasticsearch import Elasticsearch
+        hosts = []
+        for host in cfg.elasticsearch.get("hosts", "127.0.0.1:9200").split(","):
+            if host.strip():
+                hosts.append(host.strip())
+        cuckoo_index = cfg.elasticsearch.get("index", "cuckoo")
+        try:
+            # Get all indexes stored in ES
+            # Then drop all starting with cuckoo index
+            # prefix (clears previously rotated indexes)
+            es = Elasticsearch(hosts)
+            indexes = es.indices.get_aliases().keys()
+            for index in indexes:
+                if index.startswith("%s-" % cuckoo_index):
+                    es.indices.delete(index=index, ignore=[400, 404])
```

Comment:
No real reason for it, was just using it to get the names of the cuckoo indexes. Updated the commit to use the regex delete you suggested.